manual/manual/refman/patterns.etex

```diff
@@ -176,3 +176,25 @@ The pattern
 where \nth{c}{1}, \nth{c}{2}, \ldots, \nth{c}{n} are the characters
 that occur between \var{c} and \var{d} in the ASCII character set. For
 instance, the pattern "'0'"@'..'@"'9'" matches all characters that are digits.
+
+\subsubsection*{Exception patterns} \label{s:exception-match}
+(Introduced in OCaml 4.02)
+
+A new form of exception patterns, @ 'exception' pattern @ is allowed,
+only as a toplevel pattern under a "match"..."with" pattern-matching
+(other occurrences are rejected by the type-checker).
+
+Cases with such a toplevel pattern are called ``exception cases'',
+as opposed to regular ``value cases''.  Exception cases are applied
+when the evaluation of the matched expression raises an exception.
+The exception value is then matched against all the exception cases
+and re-raised if none of them accept the exception (as for a
+"try"..."with" block).  Since the bodies of all exception and value
+cases is outside the scope of the exception handler, they are all
```

Comment:
The verb must agree with the subject, which is plural, so "Since the bodies of all exception and value cases _are_ outside the scope of the exception handler" is better.